core/blockchain.go

```diff
@@ -2432,58 +2132,11 @@ func (bc *BlockChain) CXMerkleProof(toShardID uint32, block *types.Block) (*type
 	return proof, nil
 }
 
-// LatestCXReceiptsCheckpoint returns the latest checkpoint
-func (bc *BlockChain) LatestCXReceiptsCheckpoint(shardID uint32) uint64 {
```

Comment:
There are no longer needed?

Reply:
Yea, checked with Chao. it's not actually used after it's firstly designed. We went to another route.